projects/uitdatabank/docs/entry-api/organizers/create.md

```diff
@@ -1,9 +1,161 @@
+>
+> To fix this, you can [add your users as contributors](../shared/contributors.md) on your organizer after creating it.
+
+## Overview
+
+You can create a new organizers by making a single HTTP request to the [`POST /organizers`](/reference/entry.json/paths/~1organizers/post) endpoint. If successful, the organizer will be created and the response will contain the organizer's id and URL which you can store to make changes to the organizer later.
+
+A request to create a new organizer with only the required fields (website must be unique), looks like this:
+
+```http
+POST /organizers HTTP/1.1
+Host: https://io-test.uitdatabank.be
+Authorization: Bearer YOUR_ACCESS_TOKEN
+Content-Type: application/json
+
+{
+  "mainLanguage": "nl",
+  "website": "https://www.publiq.be/",
+  "name": "publiq vzw"
+}
+```
+
+After creating your new organizer, Entry API will send a response that looks like this:
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+    "id": "b48563cf-600a-43fa-acfa-0d90a2278629",
+    "organizerId": "b48563cf-600a-43fa-acfa-0d90a2278629",
+    "url": "https://io-test.uitdatabank.be/organizers/b48563cf-600a-43fa-acfa-0d90a2278629",
+    "commandId": "00000000-0000-0000-0000-000000000000"
+}
+```
+
+The `id` property contains the [UUID](https://nl.wikipedia.org/wiki/Universally_unique_identifier) of your newly created organizer, which you can use in subsequent requests to fetch or update the organizer when needed. The `organizerId` is and older version of this property and is deprecated but kept around for backward compatibility.
+
+The `url` property contains the complete URL to fetch or [update](update.md) the organizer using a `GET` or `PUT` request.
+
+Lastly, the `commandId` property is obsolete and may be ignored.
+
+## Try it out
+
+Enter your access token for the Entry API test environment below, choose a unique website and click the "Send API request" button to create the organizer from the example above.
+
+```json http
+{
+  url: 'https://io-test.uitdatabank.be/organizers',
+  method: "POST",
+  headers: {
+    "authorization": "Bearer YOUR_ACCESS_TOKEN"
+  },
+  body: {
+    "mainLanguage": "nl",
+    "name": {
+      "nl": "Voorbeeld Organizer"
+    },
+    "website": "YOUR_UNIQUE_WEBSITE"
+  }
+}
+```
+
+If you got a `201 Created` response, you successfully created your first organizer using Entry API. If you got an error instead, double check that your access token is valid and try again.
+
+Next, copy the `url` property and open it in your web browser to view the JSON of your new organizer in UiTdatabank (accessible without any authentication).
+
```

Comment:
Nit: I would add a screenshot here of your web browser where you view the JSON of an organizer, to be consistent with the docs for creating events & creating places